src/lighteval/tasks/prompt_manager.py

```diff
@@ -205,7 +206,10 @@ def _single_turn_context(
             system_prompt=system_prompt,
             use_chat_template=use_chat_template,
         )
-        toks = self.model.tok_encode(output)
+        if not use_chat_template:
+            toks = self.model.tok_encode(output)
+        else:
+            toks = "".join([msg["content"] for msg in output])
```

Comment:
Why did you change the logic here? We tokenize to make sure we won't go above allowed length